src/macaron/policy_engine/__main__.py

```diff
@@ -0,0 +1,187 @@
+# Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+"""
+Stand-alone policy engine.
+
+This program runs souffle against a macaron output sqlite database.
+"""
+
+import argparse
+import logging
+import os
+import sys
+import time
+from typing import Never
+
+from sqlalchemy import MetaData, create_engine
+
+from macaron.policy_engine.souffle import SouffleError, SouffleWrapper
+from macaron.policy_engine.souffle_code_generator import (
+    SouffleProgram,
+    get_souffle_import_prelude,
+    project_table_to_key,
+    project_with_fk_join,
+)
+
+logger: logging.Logger = logging.getLogger(__name__)
+
+
+class Config:
+    """Policy engine configuration."""
+
+    database_path: str
+    interactive: bool = False
+    policy_id: int | None = None
+    policy_file: str | None = None
+    show_prelude: bool = False
+
+
+global_config = Config()
```

Comment:
Is this configuration used anywhere?
Also, I would use `policy_config` to avoid mixing it up with Macaron's `global_config`.

Reply:
Its not used, will delete it.

Reply:
Fixed https://github.com/oracle-samples/macaron/pull/46/commits/db11e761070bbd8c28841a19da111817ef1432ea